core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/RebindManagerImpl.java

```diff
@@ -92,6 +92,9 @@
     public static final ConfigKey<RebindFailureMode> REBIND_FAILURE_MODE =
             ConfigKeys.newConfigKey(RebindFailureMode.class, "rebind.failureMode.rebind",
                     "Action to take if a failure occurs during rebind", RebindFailureMode.FAIL_AT_END);
+    public static final ConfigKey<RebindFailureMode> ADD_CONFIG_FAILURE_MODE =
+            ConfigKeys.newConfigKey(RebindFailureMode.class, "rebind.failureMode.addConfig",
+                    "Action to take if a failure occurs when setting a config value. It could happen coercion of the value type to fail.", RebindFailureMode.FAIL_AT_END);
```

Comment:
The default for isolated problems is to `.CONTINUE` which will let Brooklyn complete the startup, but display an error message in the console and deny REST requests. Better apply the same default here.